### PR TITLE
feat: [modal migration] upvoted by

### PR DESCRIPTION
**Pull request by el**
## Changes

Also fixes issue with the close button size:

image

### CODE REVIEW

File: packages/shared/src/components/RecommendedMention.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function RecommendedMention({
           key={user.username}
           user={user}
           className={classNames(
-            'px-3 cursor-pointer',
+            'py-3 px-3 cursor-pointer',
```
Comment (rebelchris) [inline packages/shared/src/components/RecommendedMention.tsx:38]:
```suggestion
            'p3 cursor-pointer',
```

**Review by rebelchris — CHANGES_REQUESTED** [review]
Some more generic to this one feedback:

- Pressing outside the modal doesn't close it for some reason
- When the list has the preloaders it actually shifts the layout (height of the modal)

**Comment by el** [comment]
@rebelchris @sshanzel here is how it looks now:

image

File: packages/shared/src/components/RecommendedMention.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/RecommendedMention.tsx]: Upps, it should be `p-3`

**Review by rebelchris — COMMENTED** [review]
It almost looks too big now compared to the one we have?
Also can we prioritised the backdrop close before pushing this live? (I don't want this to go live as it has high impact on users)

**Pull request merged**